doc/nrf/known_issues.rst

```diff
@@ -570,14 +571,14 @@ KRKNWK-12115: Simultaneous commissioning of many devices can cause the Coordinat
   #. To increase the scheduler queue size, replace ``XYZ`` next to ``ZB_CONFIG_SCHEDULER_Q_SIZE`` with the value of your choice, ranging from ``48U`` to ``256U``.
   #. To increase the buffer pool size, replace ``XYZ`` next to ``ZB_CONFIG_IOBUF_POOL_SIZE`` with the value of your choice, ranging from ``48U`` to ``127U``.
 
-.. rst-class:: v1-9-1 v1-9-0 v1-8-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0 v1-8-0
```

Comment:
@maciejpietras , input needed for this one, as it is missing from https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+2.0.0

Reply:
Sill valid for NCS2.0.0